app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordRepository.kt

```diff
@@ -19,6 +19,10 @@ class TeleconsultRecordRepository @Inject constructor(
     return teleconsultRecordDao.getCompleteTeleconsultLog(teleconsultRecordId)
   }
 
+  fun getPatientTeleconsultRecord(patientUuid: UUID, limit: Int): TeleconsultRecord? {
```

Comment:
We can create a new commit for this called `Add method to get patient latest teleconsult record` or something like that. Also please add a test for this.